packages/models-library/src/models_library/api_schemas_storage/export_data_async_jobs.py

```diff
@@ -23,5 +15,5 @@ class InvalidFileIdentifierError(StorageRpcBaseError):
 
 class AccessRightError(StorageRpcBaseError):
     msg_template: str = (
-        "User {user_id} does not have access to file {file_id} with location {location_id}"
+        "User {user_id} does not have access to one or multiple files in {paths_to_download} with location {location_id}"
```

Comment:
as an aside, if I am a user, it would make sense to tell me which file(s) are causing the issue

Reply:
Reverted and adapted code to use this format